org.sonarlint.eclipse.core/pom.xml

```diff
@@ -201,5 +201,18 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>macARM64</id>
+      <activation>
+        <os>
+          <family>mac</family>
+          <arch>aarch64</arch>
+        </os>
+      </activation>
+      <properties>
+        <protobuf.version>3.14.0</protobuf.version>
```

Comment:
Should be changed globally, not only for one OS

Reply:
I created a separate PR: https://github.com/SonarSource/sonarlint-eclipse/pull/399